test/unit/contentChunk.js

```diff
@@ -307,11 +318,21 @@ describe('contentChunk: ', () => {
       td.request.bodyId = requestBodyId
       td.response.bodyId = responseBodyId
 
+      td.orchestrations = [{ request: {}, response: {} }]
+      td.routes = [{ request: {}, response: {}, orchestrations: [{request: {}, response: {} }] }]
```

Comment:
Same as the above comment about structuring the object a bit cleaner so that its easier to follow